### ISSUE TITLE
Duplicate Meta Descriptions In Build Output

### ISSUE THREAD

**Issue by loonpwn** [vuejs/vitepress#194]
**Describe the bug**

When you set a custom meta description within the config.js and set a description at a page level using frontmatter, the build output has 2 meta descriptions

**To Reproduce**

Add a description in the vitepress config

```js
// .vitepress/config.js

module.exports = {
  // ...
  description: 'Hey 👋 I&#39;m building Laravel &amp; Vue projects and would like to share my journey with you.',
  // ...
}
```

Add a description for a page

```md
// about/index.md

---
title: About
description: "My name's Harlan. I am a full stack developer living and working out of Sydney, Australia."
head:
- - meta
  - name: description
    content: "My name's Harlan. I am a full stack developer living and working out of Sydney, Australia."
---

# {{ $page.title }}

Hey! Thank you for checking out my site 😊
```

Live example: https://harlanzw.com/about/

**Expected behavior**

A single meta description should be present, with the description from the page (about.md) taking precedence over the vitepress config description.

**System Info**
- vitepress version: 0.10.4
- vite version: 2.0.0-beta.1
- Node version: 14.15.1
- OS version: Linux Mint 20

**Comment by matias-capeletto** [comment]
There is a fix for this issue in this PR https://github.com/vuejs/vitepress/pull/170

**Comment by kiaking** [comment]
This is already fixed via #170 🙌
